exonum/src/api/websocket.rs

```diff
@@ -86,8 +91,12 @@ impl Handler<Broadcast> for Server {
     type Result = ();
 
     fn handle(&mut self, Broadcast { block_hash }: Broadcast, _ctx: &mut Self::Context) {
+        let snapshot = self.service_api_state.snapshot();
+        let schema = Schema::new(snapshot);
+        let block_header = schema.blocks().get(&block_hash);
+        let block_header_json = serde_json::to_value(block_header).unwrap().to_string();
         for address in self.subscribers.values() {
-            let _ = address.do_send(Message(format!("Committed new block {:?}", block_hash)));
+            let _ = address.do_send(Message(block_header_json.clone()));
```

Comment:
`.clone()` is confusing me a little. There is really no possibilities to get rid of it ?

Reply:
I'm not sure that's possible, as I can't move it on every loop iteration